Package.swift

```diff
@@ -34,7 +34,8 @@ let package = Package(
         .package(name: "AutoRestSwaggerBatHeader", path: "./test/integration/generated/header/"),
         .package(name: "AutoRestSwaggerBatArray", path: "./test/integration/generated/body-array/"),
         .package(name: "AutoRestBoolTest", path: "./test/integration/generated/body-boolean/"),
-        .package(name: "AutoRestUrlMutliCollectionFormatTest", path: "./test/integration/generated/url-multi-collectionFormat/")
+        .package(name: "AutoRestUrlMutliCollectionFormatTest", path: "./test/integration/generated/url-multi-collectionFormat/"),
```

Comment:
Nit:  AutoRestUrl**_Mutli_**CollectionFormatTest  should be Multi

Reply:
I can't get that. That is from the generated Package.swift  
![image](https://user-images.githubusercontent.com/8701731/99734860-6a223180-2a78-11eb-95ab-29feb5d52f0f.png)

And the typo is in the test swagger too "   "title": "AutoRest Url Mutli Collection Format Test Service","